CMakeLists.txt

```diff
@@ -75,7 +75,7 @@ if(BUILD_DEPENDENCIES)
    build_dependency(gtest)
   endif()
 
-  build_dependency(kvspic)
+  # building kvsCommonLws also builds kvspic
```

Comment:
If this is the case, we wouldnt need libkvspic-CMakeLists.txt right?

Reply:
removed